packages/commons/src/pdf-generator/pdfGenerator.ts

```diff
@@ -12,12 +12,15 @@ export interface PDFGenerator {
   ) => Promise<Buffer>;
 }
 
-export async function initPDFGenerator(): Promise<PDFGenerator> {
-  const templateService = buildHTMLTemplateService();
-  let browserInstance = await puppeteer.launch({
-    /* NOTE 
-      those configurations allow link (file://) usages for 
-      resources files in template's folder
+export const launchBrowser = (
```

Comment:
Created this function to launch puppeteer in a standard way around all the project, and reusing it everywhere we were calling puppeteer.launch